package.json

```diff
@@ -23,7 +23,8 @@
     "watch": "lighthouse-core/scripts/run-mocha.sh --watch",
     "chrome": "lighthouse-core/scripts/launch-chrome.sh",
     "fast": "npm run start -- --disable-device-emulation --disable-cpu-throttling --disable-network-throttling",
-    "smokehouse": "node $__node_harmony lighthouse-cli/test/smokehouse/smokehouse.js"
+    "smokehouse": "node $__node_harmony lighthouse-cli/test/smokehouse/smokehouse.js",
+    "deploy-viewer": "gulp --cwd ./lighthouse-viewer/ deploy"
```

Comment:
not sure. I have a slight preference for a `cd lighthouse-viewer && gulp deploy` but i dont know that they're functionally different.